.scalafix.conf

```diff
@@ -0,0 +1,7 @@
+rules = [
+    OrganizeImports
+]
+
+OrganizeImports {
+  preset = INTELLIJ_2020_3
+}
```

Comment:
I chose this configuration because although VS Code users can `Organize Imports` straight from their editor, there will always be folks using IntelliJ which has its own way of handling import organisation, so this seems like a good choice to minimise friction for contributors.

Reply:
I might otherwise agree, but the IntelliJ preset's [correctness issue](https://github.com/liancheng/scalafix-organize-imports#210-preset) concerns me.  And since most projects I maintain end up with maintenance branches, I appreciate the care the defaults put into diff reduction.

I won't die on this hill, though.